docs/src/passing-in-data.rst

```diff
@@ -0,0 +1,190 @@
+.. Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0
+   International License (the "License"). You may not use this file except in compliance with the
+   License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+   either express or implied. See the License for the specific language governing permissions and
+   limitations under the License.
+
+.. _passing_in_data:
+
+##########################################
+Passing in External Data to Your |cdk| App
+##########################################
+
+.. See https://github.com/awslabs/aws-cdk/issues/603 (includes work from the following PR)
+       https://github.com/awslabs/aws-cdk/pull/183
+
+There are a number of ways you can get data into your app from outside your app.
+
+- Using a command-line context variable
+- Using an environment variable
+- Using an SSM Parameter Store variable
+- Using a value from another stack
+- Using a |CFN| template value
+
+Each of these techniques is described in the following sections.
+
+.. _passing_in_data_from_command_line:
+
+Getting a Value from a Command-Line Context Variable
+====================================================
+
+To pass the value of a command-line context value to your app,
+run a |toolkit| command using the **--context** (**-c**) option,
+as shown in the following example.
+
+.. code-block:: sh
+
+    cdk synth -c bucket_name=mygroovybucket
+
+To get the value of a command-line context value in your app,
+use code like the following,
+which gets the value of the context variable **bucket_name**.
+
+.. code-block:: ts
+
+    const bucket_name string = this.getContext("bucket_name");
```

Comment:
This is not compiling (you need `: string`)
Also, `bucket_name` is not idiomatic in javascript, use `bucketName`.

Reply:
I'll fix.